chain/x/zoracle/internal/types/request.go

```diff
@@ -4,12 +4,12 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-type ResolveStatus int8
+type ResolveStatus string
```

Comment:
I don't think it's a good idea to save string to store. `uint8` is good as is. If you want event to be string, then create a function that converts enum to string.